lib/screens/about_page.dart

```diff
@@ -3,8 +3,15 @@ import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:smarthome/helper/theme_manager.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:github/github.dart';
+import 'package:version/version.dart';
 
 class AboutScreen extends StatelessWidget {
+  static final Version version = new Version(1, 1, 5);
+  final GitHub gitHub = new GitHub();
+  final RepositorySlug repositorySlug =
+      new RepositorySlug("susch19", "SmartHome");

```

Comment:
You already know the new bit ;)
```suggestion
      RepositorySlug("susch19", "SmartHome");

```